### PR TITLE
setup-project

### PR DESCRIPTION
**Pull request by vic778**
in this part i have
- Initialized the project and config Redux.
- Create the Header that contains the nav bar
- Add React Router and set two <Route>s and <Link>s for the app's navigation

### CODE REVIEW

File: src/pages/Books.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import BookContainer from '../components/Bookstore';
+import Header from '../components/Header';
+import InputBook from '../components/InputeBook';
+
+const Books = () => (
+  <div>
+    <Header />
+    <div style={{ padding: '10px 50px' }}>
+      <BookContainer />
```
Comment (TedLivist) [inline src/pages/Books.js:9]: - [ ] Your **`BookContainer`** should take in book data as a prop and render its `title` and `author` (or category here). Kindly fix this.
- [ ] It should also contain a placeholder **Remove** button for removing books. Kindly add it

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @vic778,

Good job so far!
✔️ Descriptive documentation
✔️ React routes are installed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
**_Check the comments under the review._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Books.js (hunk 1 above)
Comment (vic778) [reply src/pages/Books.js:9]: Thank you for the feedback but I just read and reread the description of the project I see nowhere where I should use props to display the title and the author. that being said I looked in your repo also you had not done as you had asked. however, what you ask is for tomorrow in Add and Remove a book.
please read the Microverse description [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md)

Comment (TedLivist) [reply src/pages/Books.js:9]: @vic778 I have read it. Please, do not hesitate to highlight what is unclear. As stated in the requirements, the **_building blocks_** of your app should be re-usable components (renders dynamic UI based on prop data) and books should be listed.

There should be a reusable component representing a single book and that component should take in book data as props. And it should have a Remove button (no implementation yet). Just a placeholder button. You have no such button.

**Review by Lameck1 — APPROVED** [review]
Hi, @vic778,

Well done working on the concerns of the previous review 👏🏿.
Your project is complete!  It's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**